src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -443,4 +448,18 @@ private void putCustomCapabilities(Object keyCustomCapabilities) {
     private void putCustomCapabilities(Map<String, Object> customCapabilities) {
         this.customCapabilities.putAll(customCapabilities);
     }
+
+    public SauceOnDemandAuthentication getAuthentication() {
```

Comment:
We should rename this to `getSauceAuthentication()` or something along those lines, it's not super obvious what `getAuthentication()` is for.

Reply:
good point